defaults/main.yml

```diff
@@ -301,6 +315,35 @@ tinc__etc_services__dependent_list: '{{ lookup("template",
 tinc__ferm__dependent_rules: '{{ lookup("template",
                                  "lookup/tinc__ferm__dependent_rules.j2",
                                  convert_data=False) | from_yaml }}'
+
+                                                                   # ]]]
+# .. envvar:: tinc__persistent_paths__dependent_paths [[[
+#
+# Configuration for the debops.persistent_paths_ Ansible role.
+tinc__persistent_paths__dependent_paths:
+
+  '50_debops_tinc':
+    by_role: 'debops.tinc'
+    paths: |
+      {{ [
+        '/etc/tinc',
+        '/etc/systemd/system/tinc.service',
+        '/etc/systemd/system/tinc@.service',
+        '/etc/systemd/system/multi-user.target.wants/tinc.service',
+      ] + ((ansible_local.tinc.networks.keys() | map("regex_replace", "^", "/etc/default/tinc-") | list)
+           if (ansible_local|d() and ansible_local.tinc|d() and
+               ansible_local.tinc.networks|d())
+           else [])
+      }}
+
+      ## Note that when the same network gets deleted and then added again to
+      ## `tinc__combined_networks`, the role might need two runs to also update
+      ## the defaults file in the persistent location.
+
+      ## '/etc/systemd/system/multi-user.target.wants/tinc.service':
+      ## Note that bind-dirs in Qubes OS currently does not restore symlinks (only their destination).
+      ## This works for ypid_ as he does not want auto start on Qubes OS AppVMs anyway.
+      ## If you need it on Qubes OS, feel free to discuss and patch bind-dirs.
```

Comment:
Are these comments save to parse by Ansible? I would have thought that they should be the main file comments, not comments included in the YAML text block.